package.json

```diff
@@ -59,9 +59,13 @@
     "web"
   ],
   "dependencies": {
+    "broccoli-funnel": "^1.2.0",
+    "broccoli-merge-trees": "^1.2.0",
```

Comment:
The consuming apps will need these to build the addon's vendor tree. If they're in `devDependencies`, they won't be available to the app at build-time.